scripts/tidy.sh

```diff
@@ -50,6 +50,7 @@ go_mod_update() (
     go mod tidy -compat=1.18
     go mod edit -dropreplace "github.com/pulumi/pulumi/sdk/v3"
     go mod edit -dropreplace "github.com/pulumi/pulumi/pkg/v3"
+    go mod tidy -compat=1.18
```

Comment:
I wish this weren't necessary, but to get the current version of pkg or sdk into go.sum while the replace directive is there, we need to run `go mod tidy` again.